glean-core/android/src/main/java/mozilla/telemetry/glean/acmigration/GleanACDataMigrator.kt

```diff
@@ -10,6 +10,10 @@ import androidx.annotation.VisibleForTesting
 import com.sun.jna.StringArray
 import java.lang.NullPointerException
 
+// The name of the Glean AC package, used to build the name of the files
+// of the preferences files that contain data to be migrated.
+private const val GLEAN_AC_PACKAGE_NAME = "mozilla.components.service.glean"
```

Comment:
Do we need this constant anywhere outside or should we just move it down into the companion object?

Reply:
No, we don't. Let me make it private inside the companion object